updated changelog (#121)
mapbox/mapbox-vision-android: CHANGELOG.md

# Changelog

## v0.4.1

### Vision

> - Changed behaviour on simultaneous `VisionManager` and `VisionReplayManager` instances creation to throwing an exception

Reviewer: Add `Fixed possible crash on VisionSafety/VisionArManager create/destroy`?

Reviewer 2: Agree, we've put it like 
```
- Fixed a crash that may happen on creating or destroying `VisionARManager` or `VisionSafetyManager`
```

Author: done

> - Fixed `ArLane` in Replay mode

Reviewer: Be more descriptive? Eg. `Fixed 'ArLane' not displayed in VisionReplayManager`?

Author: done

Reviewer 2: I think the reason for incorrect display of `ARLane` was incorrect values of `ARCamera` which could also affect the users. I would focus on this change and maybe mention lane issues as the consequence. In my understanding `ARLane` values were correct.

Author: make sense

# Changelog

## v0.4.1

### Vision
- Changed behaviour on simultaneous `VisionManager`
 and `VisionReplayManager` instances creation to throwing an exception

> - Fixed Replay mode obtained wrong camera-params (reason of incorrect work `ArLane`)

Reviewer 2: ```suggestion
- Fixed wrong `CameraParams` in replay mode (reason for incorrect AR lane display)
```
